tests/fixtures/estimators.py

```diff
@@ -123,3 +128,8 @@ def tmp_open_estimator_pickle(path: Optional[pathlib.Path] = None):
         return f
 
     return tmp_open_estimator_pickle
+
+
+@pytest.fixture()
```

Comment:
Since this fixture is only used in the TestRareFeatureEncoder class, it would be better to have the fixture defined in that class - then it can also be class scoped